lib/commands.py

```diff
@@ -74,10 +74,7 @@ def _ssh(hostname_or_ip, cmd, check, simple_output, suppress_fingerprint_warning
         opts.append('-o "LogLevel ERROR"')
         opts.append('-o "UserKnownHostsFile /dev/null"')
 
-    if isinstance(cmd, str):
-        command = cmd
-    else:
-        command = " ".join(cmd)
+    command = cmd if isinstance(cmd, str) else ' '.join(cmd)
```

Comment:
Why does ruff care at all about whether we use a simple if/else or a ternary?

Reply:
not sure, but we don't have to use that specific rule